conv.go

```diff
@@ -92,26 +107,45 @@ func (conv *Conv) parseSource() (float64, error) {
 	return totalSeconds, nil
 }
 
+// formatTarget converts a duration in seconds to a human-readable
+// string representation using the specified time units.
+//
+// Parameters:
+//   - seconds: The duration to format, expressed in seconds.
+//   - units: A slice of strings representing the desired time units for the output.
+//     These should correspond to keys in the unitMap (e.g., "hour", "minute", "second").
+//
+// The function iterates through the provided units, converting the input seconds into
+// each unit as appropriate. It builds a string representation, including only non-zero
+// values. The function handles singular and plural forms of the units.
+//
+// Returns:
+//   - string: A formatted string representing the duration using the specified units.
+//     For example: "2 hours 30 minutes 45 seconds"
 func (conv *Conv) formatTarget(seconds float64, units []string) string {
 	result := ""
 	for _, unit := range units {
 		unitInSeconds := unitMap[strings.TrimSuffix(unit, "s")]
```

Comment:
Shouldn't you replace all call to femkvd trailing s, by this? It's the expected way in Go

Reply:
Can you please further explain? I don't understand your comment.

Reply:
I changed this LoC to:

```go
unit := strings.ToLower(removeTrailingS(parts[i+1]))
```

I also made this change:

```go
func removeTrailingS(s string) string {
	return strings.TrimSuffix(s, "s")
}
```

In the future, `removeTrailingS` could be modified for non-English languages.